content/en/security/application_security/enabling/serverless.md

```diff
@@ -216,9 +219,767 @@ To install and configure the Datadog Serverless Framework plugin:
 
 [3]: https://app.datadoghq.com/security/appsec?column=time&order=desc
 
+{{% /tab %}}
+{{% tab "Datadog CLI" %}}
+
+The Datadog CLI modifies existing Lambda function configurations to enable instrumentation without requiring a new deployment. It is the quickest way to get started with Datadog's serverless monitoring.
+
+If you already have tracing enabled on your functions, run the following command with the `--appsec` flag to enable threat detection.
+
+**Note**: Instrument your Lambda functions in a development or staging environment first. If the instrumentation result is unsatisfactory, run `uninstrument` with the same arguments to revert the changes.
+
+To instrument your Lambda functions, run the following command:
+
+ ```sh
+  datadog-ci lambda instrument --appsec -f <functionname> -f <another_functionname> -r <aws_region> -v {{< latest-lambda-layer-version layer="python" >}} -e {{< latest-lambda-layer-version layer="extension" >}}
+```
+  - Replace `<functionname>` and `<another_functionname>` with your Lambda function names. </br>
+  - Alternatively, you can use `--functions-regex` to automatically instrument multiple functions whose names match the given regular expression.
+  - Replace `<aws_region>` with the AWS region name.
+
+Additional parameters can be found in the [CLI documentation][2].
+
+If you are configuring initial tracing for your functions, follow these steps:
```

Comment:
Can we format this to be a bit more prominent? it's not clear right now that these are the steps to follow if a customer already does NOT have APM enabled